src/api/authorization.js

```diff
@@ -0,0 +1,37 @@
+import axios from "axios"
+
+const submitAutorization = (values, NOTES, SHARED, store, userId) => {
+    axios
+      .get("https://mocki.io/v1/6e70ca5e-cb79-4b2f-8c99-8b99b08eb542")
+      .then((result) => {
+        const USERS = result.data
```

Comment:
In this context, we're getting value. So it should be in camalecase (getters/setters/calculations - > camelcase)